examples/bfetch_explorer/public/plugin.tsx

```diff
@@ -24,12 +25,16 @@ export interface BfetchExplorerStartPlugins {
   bfetch: BfetchPublicStart;
 }
 
+interface ExampleResponse extends BatchResultBase {
```

Comment:
what's this one for ?

Reply:
@ppisljar `bfetch` responses need to extend the `BatchResultBase` to have correct typescript

Reply:
why do we have example response on our public contract? (sounds like something for tests?)

Reply:
It's an example plugin. Am I missing something?

Reply:
oh :) no i was missing the part that this is example plugin. sorry.